src/models/msg.js

```diff
@@ -47,6 +47,8 @@ class Msg {
 			this.type !== Msg.Type.BANLIST &&
 			this.type !== Msg.Type.IGNORELIST &&
 			this.type !== Msg.Type.TOPIC_SET_BY &&
+			this.type !== Msg.Type.CTCP &&
```

Comment:
Instead of filtering them here, just remove the `default` case in the text logger, then it won't get written, but will remain in sqlite.